### ISSUE TITLE
workplace sites behind worplace VPN get Blocked

### ISSUE THREAD

**Issue by sbatchelder** [zen-browser/desktop#3301]
### Captchas

- [X] I have read the instructions.
- [X] I have searched existing issues and avoided creating duplicates.
- [X] I am not filing an enhancement request.

### What happened?

I am trying to access sites that are behind my work VPN (Global Protect). 

When I try to do so, I get a Server Not Found page. 
Opening internal sites on Firefox works fine, it's only on Zen that the sites fail to load.

The Networking tab for  Status on hover says: Blocked. 
![image](https://github.com/user-attachments/assets/654d993e-c526-4b77-ad0e-2409089f0474)

I tried setting adding "whoi.edu" to "Website Exceptions for DNS over HTTP". I tried turning DNS over HTTPS *OFF*. I tried adding the site to "Exceptions for Enhanced Tracking Protection". For all of these I'd also turned off uBlock Origins for the site. I'm at a loss.

I'd love to use Zen as my regular browser but the inability to access any of my workplace's internal-only sites makes that a non-starter.

### Reproducible?

- [X] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

1.0.1

### What platform are you seeing the problem on?

Linux

**Comment by sbatchelder** [comment]
This issue is still relevant on version 1.0.2-b.5

The snapshot below shows the error on Zen for a simple site behind my company firewall, contrasted with the non-error on Firefox. Two notable details are
- Zen claims the site is not encrypted, while firefox does (https)
- Zen says the site is "Verified by: Not specified", while for firefox the site is "Verified by: Internet2" 
- The site is blocked on Zen due to `NS_ERROR_UNKNOWN_HOST` error, and not blocked on Firefox

![Image](https://github.com/user-attachments/assets/b9fa755c-1a46-4a04-8959-b757d113c559)

**Comment by scarlett-blake** [comment]
Having the same issue here, but with http:// sites behind workplace VPN (using openconnect) and firewall. Likewise:
- I'm on linux and Zen version 1.0.2-b.5
- turning off DNS over HTTPS in settings has no effect
- page loads correctly in firefox (and other browsers)
- zen throws `NS_ERROR_UNKNOWN_HOST`

Happy to provide more details if it helps.

**Comment by jrgn9** [comment]
Same problem here.

- Linux with Zen 1.7b (Firefox 134.0.1) - Flatpak version
- Turning off DNS over HTTPS in settings has no effect
- Using OpenConnect version v9.12 (with AnyConnect useragent)
- Page loads correctly in Firefox (and other browsers)
- Zen throws `NS_ERROR_UNKNOWN_HOST`

**Comment by Stannieman** [comment]
I have the same issue (OpenVPN in my case). Except that sometimes it starts working, but I have not figured out what the mechanism is, and it is hard to diagnose because once it works I also don't know how to break it again for further testing.

**Comment by auggie246** [comment]
I am having the exact same issue. Only occurring to some sites behind my workplace network

**Comment by joesri** [comment]
Hi. When using a VPN, first check you 'Use system proxy settings':
In Settings->General->Network Settings, select 'Use system proxy settings'
Then, check you set 'DNS over HTTPS' to 'Off':
In Settings->Privacy & Security->DNS over HTTPS, select 'Off'

**Comment by sbatchelder** [comment]
I have Zen installed on two computers right now, both via flatpack, both updated earlier this morning. 
On my PopOS computer, sites ARE accessible thru my work VPN sites with Zen. 
On my LinuxMint computer (see screenshots from my previous posts above), even with @joesri proposed settings, I still cannot access my work VPN sites with Zen (despite being accessible with Firefox, also without having to tweak settings).

**Comment by jrgn9** [comment]
@joesri I tried to add "use system proxy settings" now, but still the same issue. It has never been an issue in Firefox. I have not even had to tweak anything, it works out of the box.

**Comment by Stannieman** [comment]
For me the proxy setting worked. Don't know if I'm just on a lucky streak though.

**Comment by Stannieman** [comment]
What I found so far:
- Some WordPress site benind VPN always works.
- Atlassian Bamboo depends:
  - If I turn on VPN BEFORE starting Zen => It works.
  - If I turn on VPN AFTER starting Zen => It doesn't work.
 
The proxy setting does not matter, it behaves like this for both system settings and no proxy.

**Comment by jrgn9** [comment]
For what its worth, I tested Floorp (another Firefox based browser) and I have the same problem with VPN there. Seems like I have to stick with vanilla Firefox and Vivaldi for the time being.

**Comment by mr-cheffy** [comment]
## Closing this issue as part of a repo cleanup

Due to technical-debt we had in the past, we are currently clearing out old, duplicate, stale, and non-actionable issues to improve issue tracking and make it easier to focus on fixing real bugs.

If this issue is still relevant, please open a new issue using our updated issue template, which will help us categorize and address it more efficiently.

Thanks for your understanding and for helping us improve the project! 🚀

This issue can be reopened at: https://github.com/zen-browser/desktop/issues/new?template=bug_report.yml&title=workplace%20sites%20behind%20worplace%20VPN%20get%20Blocked%0A

**Comment by iNikAnn** [comment]
Installed Zen v1.21.15b (Flatpak) on Ubuntu 26.04.

"Use system proxy settings" doesn't work. Traffic bypasses proxy completely.

**Fix:** Go to `Settings` -> `Privacy and security` -> `Connection and software security` -> `Proxy Settings`, select "Manual proxy configuration", enter host and port manually (e.g. 127.0.0.1:7890).
